CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 CHANGELOG
 =========
 
+2.3 (2022-xx-xx)
```

Comment:
Change this title to "unreleased".

We can only define the new version at the point we're ready to release.